### ISSUE TITLE
How to modify bios firmware image version?

### ISSUE THREAD

**Issue by Raj2032** [LongSoft/UEFITool#396]
I want to go back to a previous version of my bios however my current bios version does not let me.

I want to hack the bios file that I downloaded

https://www.gigabyte.com/Motherboard/GA-Z97X-SOC-rev-10/support#support-dl-bios

and change its version number, that way when I go to update the bios version, it will think it is a new version rather than an older version.

How would I do this? Which field am I looking for?

**Comment by NikolajSchlej** [comment]
I do not think it makes sense to do so, as the version is often hardcoded in many places, and some information is also derived from it, that would then also need to be replaced. If you want to revert to a given BIOS version, there are many other ways, including USB BIOS Flashback (or whatever it is called on Gigabyte motherboards), SPI flash programmer, removing BIOS Lock using NVRAM editing, and so on.

**Comment by Raj2032** [comment]
yep already done now.
